src/types/profile.js

```diff
@@ -131,7 +131,8 @@ export type RawMarkerTable = {|
  * time. The relationship between frames is defined by the StackTable.
  */
 export type FrameTable = {|
-  address: IndexIntoStringTable[],
+  // The address is a copy from the FuncTable entry.
+  address: Array<MemoryOffset | -1>,
```

Comment:
Not for this PR: I wonder if we should change this to `null` instead of `-1`?

Reply:
I thought of this too, I believe this is possible.